bitnami/external-dns/values-production.yaml

```diff
@@ -411,6 +411,8 @@ rbac:
   ## Service Account for pods
   ## https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/
   ##
+  serviceAccount:
+    crete: true
   serviceAccountName: default
```

Comment:
Suggestion: `serviceAccountName` should be undefined by default. This way, you can simply check in the rest of the templates whether it's defined or not:
```diff
-
```